doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -340,6 +341,11 @@ Bluetooth Fast Pair samples
   * Updated the :ref:`fast_pair_locator_tag_testing_fw_update_notifications` section to improve the test procedure.
     The application provides now an additional log message to indicate that the firmware version is being read.
 
+Bluetooth nRF_Auraconfig sample
+-------------------------------
+
+ * Buffer count (CONFIG_BT_ISO_TX_BUF_COUNT) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use-cases.
```

Comment:
Move under Bluetooth samples, as this sample does not have its own section under Samples.
After moving, change entry to following:
```suggestion
* :ref:`nrf_auraconfig` sample:

  * Updated the buffer count (:kconfig:option:`CONFIG_BT_ISO_TX_BUF_COUNT`) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use cases.
```

Reply:
In addition to Greg's suggestion, move the second sentence to the new line:
"This can be changed and optimized for specific use cases."